packages/cubejs-cockroach-driver/docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: '3'
+
+services:
+
+ crdb:
+   container_name: ${CONTAINER_NAME}
+   image: cockroachdb/cockroach:v22.1.1
```

Comment:
🎵 

just take the latest one

Reply:
Would it be better to use the `:latest` for this?